src/inversify.config.ts

```diff
@@ -41,6 +42,8 @@ export const createRootContainer = (args: ArgumentsProvider): Container => {
   container.bind(FileSystemService).toSelf();
   container.bind(GitHubService).toSelf();
   container.bind(BitbucketService).toSelf();
+  container.bind(Types.ICollaborationInspector).to(CollaborationInspector);
+  container.bind(Types.IIssueTrackingInspector).to(IssueTrackingInspector);
```

Comment:
I guess Inspectors should be bind for a project component, not in a root container